### ISSUE TITLE
How to set reasonable parameters

### ISSUE THREAD

**Issue by vaeXu** [michalfaber/rmpe_dataset_server#4]
@michalfaber Thanks for sharing, I want to try this training but I don't know how to set parameters，what should i do ? thank you.
  params.stride=8;
  params.crop_size_x=368;
  params.crop_size_y=368;
  params.target_dist=0.6;
  params.scale_prob=1;
  params.scale_min=0.5;
  params.scale_max=1.1;
  params.max_rotate_degree=40;
  params.center_perterb_max=40;
  params.do_clahe=false;
  params.num_parts_in_annot=17;
  params.num_parts=56;
  params.mirror = true;

**Comment by michalfaber** [comment]
The defaults which you can modify are here https://github.com/michalfaber/rmpe_dataset_server/blob/b7c815c6f29eb7872424e354e5ee7710ba0da2f6/main.cpp#L137 but this repo will be soon deprecated. I work on tensorpack based mechanism for augmenting/serving dataset - all in python.

**Comment by vaeXu** [comment]
@michalfaber thanks for your quick reply.But how can I use Python to modify it? i took 12hours to  train ./rmpe_dataset_server ../../keras_Realtime_Multi-Person_Pose_Estimation/dataset/val_dataset.h5 5556,but it keep 
Epoch 1
curr_sample/total_samples/curr_epoch = 1/2645/1

what should i do? thank you.

**Comment by michalfaber** [comment]
@GAVANXU You should start training generator on port 5555 :  ./rmpe_dataset_server ../../keras_Realtime_Multi-Person_Pose_Estimation/dataset/training_dataset.h5 5555
Currently validation set is disabled but you can try to enable it https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/blob/e05cb7d3bb9da6bd2e95d2a04a146514ed87b673/training/train_pose.py#L169
